src/Teams/Startup.cs

```diff
@@ -31,7 +31,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<ApplicationDbContext>(options =>
                 options.UseSqlServer(
                     Configuration.GetConnectionString("DefaultConnection")));
-            services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true)
+            services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = false)
```

Comment:
Well done, but you can remove value assignment to RequireConfirmedAccount at all, because it will be equal to false by default.